dev/NumberBox/TestUI/NumberBoxPage.xaml

```diff
@@ -1,154 +1,253 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
-<local:TestPage
-    x:Class="MUXControlsTestApp.NumberBoxPage"
-    x:Name="NumberBoxTestPage"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:MUXControlsTestApp"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,5)"
-    mc:Ignorable="d">
+﻿<!--  Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information.  -->
```

Comment:
A lot of these changes are only formatting, please revert them.

Reply:
Sorry, XAML Styler was interfering again. Disabled it for now. 

I have reverted the changes. I should be ok again.

Reply:
I think you forgot to push that commit 😅

Reply:
I'd also prefer the styling update to be a separate commit

---
In reply to: [456551014](https://github.com/microsoft/microsoft-ui-xaml/pull/2901#discussion_r456551014) [](ancestors = 456551014)

Reply:
I forgot the Push 😒 and made now two separate commits 😊.